Lesson_2_JSON_YAML_XML/Unicorn json.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Uno",
+    "age": 2,
+    "hasOneHorn": true,
+    "favoriteFoods": ["Peaches", "Strawberry"],
+    "isInCartoons": [
```

Comment:
The naming can be a bit misleading, usually, we use keys starting with "is[Adjective]" to highlight that the value will be a Boolean

```
"isRegistered": true
"isActive": false
```

etc

Reply:
As a better option, smth like "CartoonParticipations" or "CartoonAppearences" or anything that can

a) Make someone who reads it assume it's an array/list
b) Describe the semantical purpose of a value